bin/manuscripts

```diff
@@ -62,7 +62,8 @@ def get_params():
     parser.add_argument('-p', '--projects', action='store_true', help="Generate per project data")
     parser.add_argument('--indices', default=[], nargs='*',
                         help="Indices to be used to generate the report (git_index, github_index ...)")
-
+    parser.add_argument('-l', '--logo', help="Provide a logo for the report, Formats allowed .png,.pdf,.jpg,.mps,.jpeg,.jbig2,.jb2,.PNG,.PDF,.JPG,.JPEG,.JBIG2,.JB2,.eps")
```

Comment:
Please, improve the punctuation and simplify the sentence. I suggest it to be "Logo for the report. Formats (extensions): ...."

In addition, did you see my comment about ensuring all these formats are supported. Please, let us know if you know that all those formats are supported (and why).

In addition, please say which one is the default (GrimoireLab logo), and add it as default value for `add_argument`

Reply:
@jgbarah  when you remove the default logo file from template folder , the follwing message  is thrown by pdflatex:

I could not locate the file with any of these extensions:
.png,.pdf,.jpg,.mps,.jpeg,.jbig2,.jb2,.PNG,.PDF,.JPG,.JPEG,.JBIG2,.JB2,.eps

And i didnt specify default for the argument because the latex file already has 'logo' as default logo file in it

Reply:
Sorry for the long delay.

OK to the extensions. I checked, and they seem to work, anyway.

I understand what you mean about the default logo. Maybe it would be better to have it more explicitly as a default value, but that's not a big deal, so let's go with it.